flowauth/Pipfile

```diff
@@ -23,7 +23,7 @@ pytest-cov = "*"
 "flowauth" = {editable = true, path = "./backend"}
 
 [requires]
-python_version = "3.7"
+python_version = "3.6"
```

Comment:
I'm ok to drop the version to 3.6 for flowauth for the time being. If someone wants to build/develop flowauth locally this makes it a little less convenient because the version now differs from the one for the other components, so it needs switching via pyenv or similar. However, hopefully we can switch back to 3.7+ soon. Can we open a Github issue so we won't forget to do this?